Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonColorShade.kt

```diff
@@ -8,31 +8,43 @@ enum class PersonColorShade(
     @ColorRes val starsTint: Int,
     @DrawableRes val background: Int,
     @DrawableRes val currentCertificateBg: Int,
+    @DrawableRes val bookmarkIcon: Int,
     @DrawableRes val admissionBadgeBg: Int,
 ) {
 
     COLOR_1(
         R.color.starsColor1,
         R.drawable.bg_person_overview_1,
         R.drawable.bg_certificate_blue_1,
+        R.drawable.ic_bookmark_blue_1,
         R.drawable.ic_admission_badge_1,
     ),
     COLOR_2(
         R.color.starsColor2,
         R.drawable.bg_person_overview_2,
         R.drawable.bg_certificate_blue_2,
+        R.drawable.ic_bookmark_blue_2,
         R.drawable.ic_admission_badge_2,
     ),
     COLOR_3(
         R.color.starsColor3,
         R.drawable.bg_person_overview_3,
         R.drawable.bg_certificate_blue_3,
+        R.drawable.ic_bookmark_blue_3,
         R.drawable.ic_admission_badge_3,
     ),
     COLOR_INVALID(
         R.color.starsColorInvalid,
         R.drawable.bg_person_overview_invalid,
         R.drawable.bg_certificate_grey,
+        R.drawable.ic_bookmark,
+        R.drawable.ic_admission_badge_1,
+    ),
+    COLOR_UNDEFINED(
```

Comment:
For which situation do we need undefined?

Reply:
As a default value: because we use the same fragment when navigating from UQS. Not using this COLOR_UNDEFINED would lead to additional navigation flag (because we need to define if colour was defined or not) and I don't want use it.

Reply:
Could you also use null instead of undefined? If I see it correctly, the attributes of undefined are never used.